Casks/m/mediamate.rb

```diff
@@ -0,0 +1,30 @@
+cask "mediamate" do
+  version "2.4.1,204"
+  sha256 "08808c35b2421a414ed3e69b611083c2349f48611a43519bfc2ef252e33abb51"
+
+  url "https://github.com/Wouter01/MediaMate-Releases/releases/download/v#{version.csv.first}_#{version.csv.second}/MediaMate_v#{version.csv.first}-#{version.csv.second}.zip",
+      verified: "github.com/Wouter01/MediaMate-Releases/"
+  name "MediaMate"
+  desc "Control volume, brightness, and now playing with a new look"
```

Comment:
I'm not sold on this example, but I think the `desc` can be better.

```suggestion
  desc "UI replacement for volume, brightness and now playing control"
```

Reply:
Looks good! Thanks!